src/miRBench/encoder.py

```diff
@@ -167,7 +167,7 @@ class TargetNetEncoder():
     U = [0, 0, 0, 1, 0]
     T = [0, 0, 0, 1, 0]
     - = [0, 0, 0, 0, 1]
-    First, gene sequence is truncated to 40nt nad miRNA sequence is reversed to be in 5' to 3' direction.
+    First, gene sequence is truncated to 40nt and reversed.
```

Comment:
I would add "to be in the 3' to 5' direction", in agreement with https://academic.oup.com/bioinformatics/article/38/3/671/6408435#394284742:~:text=%2C-,where,in%20the%205%E2%80%B2%2Dto%2D3%E2%80%B2%20and%203%E2%80%B2%2Dto%2D5%E2%80%B2%20directions%2C%20respectively.,-We%20use%20subscript

Reply:
Added.